nautobot_golden_config/forms.py

```diff
@@ -107,7 +110,7 @@ class Meta:
         fields = ("name", "slug", "description", "tags")
 
 
-class GoldenConfigFilterForm(NautobotFilterForm):
```

Comment:
have you tested this? I did it this way since it seems something is stripping it out.

Reply:
I fixed and tested. It looks like what I expect after my last commit